compiler/ExtractBase.ml

```diff
@@ -1390,7 +1398,7 @@ let ctx_compute_type_name_no_suffix (span : Meta.span) (ctx : extraction_ctx)
   flatten_name (ctx_compute_simple_type_name span ctx name)
 
 (** Provided a basename, compute a type name. *)
-let ctx_compute_type_name (span : Meta.span) (ctx : extraction_ctx)
+let ctx_compute_type_decl_name (span : Meta.span) (ctx : extraction_ctx)
```

Comment:
Can you add an "_aux" suffix? I now remember why this function was named like that (there is a `ctx_compute_type_decl_name` below)